tests/core/tools.js

```diff
@@ -628,6 +628,99 @@
 			assert.areSame( 'ABcDeF', c( 'aBcDeF', true ) );
 		},
 
+		'test throttle': function() {
+			var output = 0,
+				foo = 'foo',
+				buz = 'buz',
+				message,
+				buffer = CKEDITOR.tools.throttle( 200, function( arg ) {
```

Comment:
The problem with `sinon.assert` is that it doesn't allow for meaningful assert messages. because of that issue, I will use `assert.isTrue( inputSpy.calledOnce, msg )` assert.